source/mainnet/net/guides/recover-wallet.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: ../../variables.rst
+.. _recover-wallet:
+
+===================
+Recover your Wallet
+===================
+
+In case you get a new device or need to restore your existing device, you can recover your identities, accounts, and keys with the secret recovery phrase you wrote down during wallet setup.
+
+.. Note::
+
+    You cannot recover testnet wallets on mainnet, or vice versa.
+
+.. tabs::
+
+    .. tab:: |mw-gen2|
+
+        #. After reinstalling the |mw-gen2| app, open the app.
+
+        #. On the Getting Started screen, tap **Recover wallet**.
+
+            .. image:: ../images/mobile-wallet-gen2/choice-start.png
+                :width: 25%
+
+        #. After the screens explaining recovery tap **Continue**.
+
+        #. Enter each word of your recovery phrase in the correct order. When you start typing, possible words appear for you to select. Once the words are correct, tap **Continue** to submit the recovery request to the identity provider(s).
+
+        #. When recovery is successful, the screen below appears.
+
+        Sometimes recovery can take longer. Here are some scenarios that you might encounter.
```

Comment:
The scenarios below have changed. There are only two scenarios now: Either everything went fine, or the recovery was partial. If the wallet had trouble connecting or getting something meaningful from the identity provider, it will say the recovery was partial. If the ID couldn't be recovered, the related accounts will also not be recovered. The user will have to try again now or later to get the rest.